modules/host/consts.go

```diff
@@ -78,10 +78,10 @@ var (
 	defaultCollateralBudget = types.SiacoinPrecision.Mul64(100e3)
 
 	// defaultContractPrice defines the default price of creating a contract
-	// with the host. The default is set to 30 siacoins, which the file
-	// contract revision can have 15 siacoins put towards it, and the storage
-	// proof can have 15 siacoins put towards it.
-	defaultContractPrice = types.SiacoinPrecision.Mul64(3) // 3 siacoins
+	// with the host. The default is set to 0 siacoins. This should be fine
+	// since we use the FeeEstimation function for an estimation which already
+	// has a sane minimum > 0.
+	defaultContractPrice = types.ZeroCurrency // 0 siacoins
```

Comment:
default minimum of 0 will always make me nervous. Would feel better if it was 0.1 or 0.01

Reply:
Could we set it to 0.001? It is redundant due to the minimum in `FeeEstimation` but I understand that it makes you nervous :p 
While we're on it, could we decrease the sane minimum of the `FeeEstimation` by 100x? I feel like the usd price probably increased by 100x since that minimum was introduced. That would cause the actual minimum contract price to be 0.003 SC instead of 0.3 SC which is much more reasonable if you form 50-100 contracts.